src/components/phylogeny/Phylotree.vue

```diff
@@ -130,6 +141,91 @@ export default {
     this.redrawTree();
   },
   methods: {
+    exportAsNexus() {
+      // Export this phylogeny as a Nexus string in a .nex file for download.
+      const newickStr = this.tree.getNewick((node) => {
+        // Is the resolved node for this phyloref? If so, let's make an annotation.
+        if (
+          this.phyloref !== undefined &&
+          has(node, "data") &&
+          has(node.data, "@id")
+        ) {
+          const annotations = [];
+          const data = node.data;
+
+          const convertToNexusAnnotationValue = (str) => {
+            // We really just need to wrap this in double-quotes, which means we need to filter out existing double quotes.
+            return '"' + str.replaceAll('"', "''") + '"';
```

Comment:
So I take it there's no escape character? It bothers me a bit that we're substituting one character (double quote) with two other characters (two single quotes). This would look right if we output TeX, but we don't (nor is anyone necessarily expecting TeX), and it'll make for an essentially non-reversible substitution. (E.g., if a single quote were to be adjacent to a double quote, we'd have 3 single quotes and now on trying to reverse the substitution we wouldn't be able to automatically decide whether the first or the second two single quotes needed to be substituted back to a double quote.)

Reply:
True! I could replace it with a single quote if that would be better. I tried the standard quote characters (`\"`, `""`) and neither of them worked. Do you have any ideas of other quote characters I could try?

Reply:
TODO: replace with a single quote.

Reply:
TODO: Provide some sort of warning message if there are already single quotes, "This cannot be substituted back -- do you want to proceed anyway?" -- that way users will know what to expect.